frontend/src/pages/members/MemberClaimsTab.tsx

```
@@ -0,0 +1,166 @@
+                
+                <button
+                  onClick={() => navigate(`/claims/${claim.id}`)}
+                  className="ml-4 p-2 rounded-lg transition-colors"
+                  style={{ backgroundColor: colors.background.tertiary, color: colors.text.primary }}
+                  onMouseEnter={(e) => {
+                    e.currentTarget.style.backgroundColor = colors.background.hover;
+                  }}
+                  onMouseLeave={(e) => {
+                    e.currentTarget.style.backgroundColor = colors.background.tertiary;
+                  }}
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)